libs/evcs-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -430,7 +433,8 @@ void testUpdateVcRequestReturnsUsersVcWith204(MockServer mockServer) throws Exce
     @Pact(provider = "EvcsProvider", consumer = "IpvCoreBack")
     public RequestResponsePact invalidUpdateUserVcReturns400(PactDslWithProvider builder) {
         return builder.given("EVCS client exist")
-                .given("User has a valid VC")
+                .given("test-evcs-api-key is a valid API key")
```

Comment:
Does this (and any others) need `Existing user` or `Brand new user` as well?

Reply:
hm yes I thought same but they wanted to have 

`"test-user-id has one CURRENT VC"`

May be some test data based on userId